features/testbot/basic.feature

```diff
@@ -251,3 +251,34 @@ Feature: Basic Routing
             | from | to | route |
             | d    | c  | de,ce,ce |
             | e    | d  | de,de    |
+
+    Scenario: Ambiguous edge weights - Use minimal edge weight
+        Given the node map
+            | a | b |
+
+        And the ways
+            | nodes | highway   |
+            | ab    | tertiary  |
+            | ab    | primary   |
+            | ab    | secondary |
+
+        When I route I should get
+            | from | to | route | time |
+            | a    | b  | ab,ab | 10s  |
+            | b    | a  | ab,ab | 10s  |
+
+    Scenario: Ambiguous edge names - Use maximal name id
+        Given the node map
+            | a | b | c |
+
+        And the ways
+            | nodes | highway | name |
+            | ab    | primary | 1    |
+            | ab    | primary |      |
+            | ab    | primary | 2    |
+            | bc    | primary | 3    |
+
+        When I route I should get
+            | from | to | route |
+            | a    | c  | 2,3,3 |
+            | c    | a  | 3,2,2 |
```

Comment:
Could fail randomly, due to parallel parsing. `ab` are all identical, so they are differentiated on by the name_id which is random due to https://github.com/Project-OSRM/osrm-backend/blob/0e4061f85816b5499896022eb151a1402ae3aad0/src/extractor/extractor.cpp#L212 and its interaction with https://github.com/Project-OSRM/osrm-backend/blob/0e4061f85816b5499896022eb151a1402ae3aad0/src/extractor/extractor.cpp#L160